tf/environments/dev/main.tf

```diff
@@ -427,19 +427,55 @@ module "ooni_backendproxy" {
   key_name      = module.adm_iam_roles.oonidevops_key_name
   instance_type = "t3a.nano"
 
-  backend_url        = "https://backend-fsn.ooni.org/"
-  wcth_addresses     = module.ooni_th_droplet.droplet_ipv4_address
-  wcth_domain_suffix = "th.ooni.org"
-  clickhouse_url     = "clickhouse1.prod.ooni.io"
-  clickhouse_port    = "9000"
+  name = "oonickprx"
+  ingress_rules = [{
+    from_port = 22,
+    to_port = 22,
+    protocol = "tcp",
+    cidr_blocks = ["0.0.0.0/0"],
+  }, {
+    from_port = 80,
+    to_port = 80,
+    protocol = "tcp",
+    cidr_blocks = ["0.0.0.0/0"],
+  }, {
+    from_port = 9000,
+    to_port = 9000,
+    protocol = "tcp",
+    cidr_blocks = ["0.0.0.0/0"],
```

Comment:
We should only allow ingress traffic from inside of AWS

Reply:
Move to allowing ingress traffic only from the private subnet: https://github.com/ooni/devops/pull/140/files#diff-d39a66a559566d5f64edd0ff9ee12f2962643ea3b7bf4aaa8b542347b40fca6eR438, where the ECS cluster resides.